### PR TITLE
Update package release history

### PR DESCRIPTION
**Pull request by troydai**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: packaged_releases/HISTORY.md [hunk 1]
```diff
@@ -1,6 +1,59 @@
 Release History
 ===============
 
+2.0.8 (2017-06-13)
+------------------
+
+* See the `HISTORY.rst` files in each module for a changelog.  
+
+Source download:
+https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_2.0,8.tar.gz
```
Comment (derekbekoe) [inline packaged_releases/HISTORY.md]: `2.0,8` -> `2.0.8`

File: packaged_releases/HISTORY.md [hunk 2]
```diff
@@ -1,6 +1,59 @@
 Release History
 ===============
 
+2.0.8 (2017-06-13)
```
Comment (yugangw-msft) [inline packaged_releases/HISTORY.md]: @troydai, mind adding an entry for the greatest 2.0.9 azure-cli-core? Or, i can submit a PR after this get merged

Comment (troydai) [inline packaged_releases/HISTORY.md]: @derekbekoe do we need to add it here? since we didn't ship it in neither bundle or apt.

**Comment by troydai** [comment]
@derekbekoe updated.

File: packaged_releases/HISTORY.md (hunk 2 above)
Comment (derekbekoe) [inline packaged_releases/HISTORY.md]: @troydai @yugangw-msft No we wouldn't add it since we're not releasing this.

**Pull request merged**